### ISSUE TITLE
Add extension to default CLI config filename

### ISSUE THREAD

**Issue by localnet** [mikro-orm/mikro-orm#174]
**Describe the bug**
Currently, without configure anything on package.json, the CLI by default search by a config filename `cli-config` without extension.

**Expected behavior**
I think it should search by a config filename `cli-config.js` with extension.

**Additional context**
I too would suggest use, by default, a more typical filename as `mikro-orm.config.js` or `mikro-cli.config.js` (as others: Webpack: `webpack.config.js`, Babel: `babel.config.js`)

**Versions**

| Dependency | Version |
| - | - |
| node | 10.16.3 |
| typescript | 3.6.2 |
| mikro-orm | 3.0.0-alpha.16 |
| your-driver | mariadb |

**Comment by B4nan** [comment]
Yeah I am also thinking about using different default filename (`cli-config` comes from doctrine), `mikro-orm.config.js/ts` sounds good I guess. 

About the extension, the idea was that `cli-config` will resolve to both `cli-config.js` and `cli-config.ts` (that is how node's require work). I totally forgot to try this out, as I immediately used paths from `package.json`. Will fix that so it will search for both js/ts extensions.

**Comment by lookfirst** [comment]
I'm using `src/db/mikro-config.ts` cause I use the same config file for both the cli and the app.

```json
  "mikro-orm": {
    "useTsNode": true,
    "configPaths": [
      "./src/db/mikro-config.ts"
    ]
  }
```

**Comment by B4nan** [comment]
That is a name of [my first nodejs package](https://github.com/B4nan/mikro-config) :P 

The name here is MikroORM so I would prefer `mikro-orm.config`. But that is the reason why I allowed to specify the path including file name, so one can pick whatever filename suites them best.

**Comment by B4nan** [comment]
Implemented in `3.0.0-alpha.18`, the file is now called `mikro-orm.config.js`, if you enable `ts-node`, then also `mikro-orm.config.ts` is searched for (and has priority).
